numpy/linalg/linalg.py

```diff
@@ -1810,7 +1810,7 @@ def det(a):
 
 # Linear Least Squares
 
-def lstsq(a, b, rcond=-1):
+def lstsq(a, b, rcond="warn"):
```

Comment:
You could argue, NoValue means the same in the sense of "not passed in", but I just don't think we need a singleton like this here when a meaningful string will do. Just my opinion though.